app/components/Blockchain/Fees.jsx

```diff
@@ -15,7 +15,7 @@ let ops = Object.keys(operations);
 
 // Define groups and their corresponding operation ids
 let fee_grouping = {
-    general: [0, 25, 26, 27, 28, 32, 33, 37, 39, 41, 49, 50, 52, 69, 70, 71, 72, 73],
+    general: [0, 25, 26, 27, 28, 32, 33, 37, 39, 41, 49, 50, 52, 59, 60, 61, 62, 63, 69, 70, 71, 72, 73],
```

Comment:
I think LP-related fees should be in the `market` group. Thanks.

Reply:
I thought so but I saw *Credit Offer* under general; so I said I might be wrong as usual :P